test/suite/helper_tiered.py

```diff
@@ -140,7 +140,6 @@ def gen_tiered_storage_sources(random_prefix='', test_name='', tiered_only=False
         ('s3_store', dict(is_tiered = True,
```

Comment:
Line 110: Left an instance of has_cache
`            has_cache = True,`

Reply:
removed [12ff933](https://github.com/wiredtiger/wiredtiger/pull/9708/commits/12ff9339c1e48e451f1f053627295e0c4ddb4836)